javascript/happyx/config.yaml

```diff
@@ -0,0 +1,13 @@
+framework:
+  github: Hapticx/happyx
+  website: hapticx.github.io/happyx
+  version: 1.1.1
+
+  bootstrap:
+    - npm install happyx
+    - npm install
```

Comment:
no need, this is kind a standard and will be done (there is twice this command in generated dockerfile)
see `bundle exec rake config`